test/cli/test-files.js

```diff
@@ -9,14 +9,15 @@ const _ = require('lodash')
 
 describe('files', () => {
   const env = _.clone(process.env)
-  env.IPFS_PATH = repoPath
+  process.env.IPFS_PATH = repoPath
```

Comment:
This should not be modified. As `env` is passed in below into the command and this would modify the whole process.